package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@r/flags",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
🔕 In general it's probably better to do npm version updates in separate commits when you merge your pr. When there are concurrent pr's its easy to merge without really updating the version (two separate pr's could have optimistically bump'd). Plus its nice to see the version updates in the commit log.